report.20210618.141351.42711.0.001.json

```diff
@@ -0,0 +1,635 @@
+
+{
+  "header": {
+    "reportVersion": 1,
+    "event": "Allocation failed - JavaScript heap out of memory",
+    "trigger": "FatalError",
+    "filename": "report.20210618.141351.42711.0.001.json",
+    "dumpEventTime": "2021-06-18T14:13:51Z",
+    "dumpEventTimeStamp": "1624047231366",
+    "processId": 42711,
+    "cwd": "/Users/sondrae/Sites/oddleventy",
+    "commandLine": [
+      "/Users/sondrae/.nvm/versions/node/v12.16.1/bin/node",
+      "/Users/sondrae/Sites/oddleventy/.yarn/cache/chokidar-cli-npm-2.1.0-3bc4fdea14-6d174178b4.zip/node_modules/chokidar-cli/index.js",
+      "**/.stylelintrc.yml",
+      "-c",
+      "yarn lint:sass"
+    ],
```

Comment:
I'm guessing we would want to delete this file? and maybe add it to gitignore? Can you confirm @jgerigmeyer